patches/@types+jquery.scrollto+1.4.29.dev.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@types/jquery.scrollto/index.d.ts b/node_modules/@types/jquery.scrollto/index.d.ts
+index 0a00c69..1707682 100755
+--- a/node_modules/@types/jquery.scrollto/index.d.ts
++++ b/node_modules/@types/jquery.scrollto/index.d.ts
+@@ -6,7 +6,7 @@
+ 
+ /// <reference types="jquery"/>
+ 
+-interface ScrollToOptions {
++interface ScrollToOptions extends JQuery.EffectsOptions<any> {
```

Comment:
There wasn't a dep update. Did we need to update this patch?

Reply:
It's because of the code below:

https://github.com/cypress-io/cypress/blob/68173be4aa3a9d6c64cc447aa68d8f374bc3c066/packages/driver/src/cy/commands/actions/scroll.ts#L121-L143

`ScrollToOptions` uses `Jquery.EffectsOptions` methods like `done()`, `always()`, but they don't exist without extending it.